src/js/plugin.js

```diff
@@ -451,6 +451,16 @@ Plugin.BASE_PLUGIN_NAME = BASE_PLUGIN_NAME;
 
 Plugin.registerPlugin(BASE_PLUGIN_NAME, Plugin);
 
+/**
+ * Documented in player.js
+ *
+ * @ignore
+ */
+Player.prototype.usePlugin = function(name, plugin, ...options) {
+  Plugin.registerPlugin(name, plugin);
+  this[name](...options);
```

Comment:
We'd want to do a null-check around `this[name]` being a method.

Reply:
In this scenario, is it possible to be null? Wouldn't the `registerPlugin` throw instead?